main/solution/infrastructure/config/infra/cloudformation.yml

```diff
@@ -2,6 +2,12 @@ Conditions:
   IsDev: !Equals ['${self:custom.settings.envType}', 'dev']
```

Comment:
This fix already exists on `develop` now. Any reason why we needed it in this PR?

Reply:
I added it because when I was testing the `fix-rstudio-alb-pentest` branch, it's needed to create the right HostedZone for RStudio. So I think it's better to merge this fix into `fix-rstudio-alb-pentest` so the bugfix is in the feature branch as well as develop. 

It doesn't hurt to include it, and it makes development easier as it'll allow installation of SWB to proceed correctly.